backend/controllers/userController.js

```diff
@@ -0,0 +1,14 @@
+// userController.js
+const UserModel = require('../models/userModel');
+
+// Example controller function
+const getUsers = async (req, res) => {
+    try {
+        const users = await UserModel.find();
```

Comment:
How is the UserModel going to know which user to fetch  without passing any params to it

Reply:
This is just a boiler plate to set general structure of the code directories. Its a stub for all what matters.